common/src/main/java/whocraft/tardis_refined/common/tardis/manager/TardisInteriorManager.java

```diff
@@ -288,6 +302,27 @@ public void tick(ServerLevel level) {
         }
     }
 
+    public boolean shouldTheEyeBeOpen(ServerLevel level) {
+
+        return level.getBlockState(pillarTopLeft).getBlock() == TRBlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(pillarTopRight).getBlock() == TRBlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(pillarBottomLeft).getBlock() == TRBlockRegistry.ARTRON_PILLAR.get() && level.getBlockState(pillarBottomRight).getBlock() == TRBlockRegistry.ARTRON_PILLAR.get() && operator.getTardisState() != TardisLevelOperator.STATE_EYE_OF_HARMONY;
+    }
+
+    public void openTheEye() {
+        this.operator.setTardisState(TardisLevelOperator.STATE_EYE_OF_HARMONY);
+        //operator.getPilotingManager().setFuel(operator.getPilotingManager().getMaximumFuel());
+
+        // Remove the blocks
+        BlockPos.betweenClosedStream(new AABB(1011, 72, 54, 1015, 71, 56)).forEach(x -> this.operator.getLevel().setBlock(x, Blocks.AIR.defaultBlockState(), 3));
```

Comment:
Use variable to document corner positions, use Block Flags for set block instead of magic number.